src/api/app/controllers/status_controller.rb

```diff
@@ -1,29 +1,6 @@
 require_dependency 'status_helper'
 
 class StatusController < ApplicationController
-  def workerstatus
-    send_data(WorkerStatus.hidden.dump_xml)
```

Comment:
as you're touching this activexml code, would you please either wait for https://github.com/openSUSE/open-build-service/pull/5605 or cherry-pick the code that handles the worker status?

Reply:
i would prefer to do not have to wait for it and not have to touch it either.. I'm just working on a card that I would like to have it ready to the end of the sprint... is that ok @coolo ?

Reply:
I wouldn't have asked if it was ok for me.

Reply:
I am sorry, but we can not block other PRs just because of possible merge conflicts.

That said, it doesn't look like this would require many change to merge it with [your changes](https://github.com/openSUSE/open-build-service/pull/5605/files#diff-2f691a4483e387f933a50dede911b8d9L3). If this turns out to be more effort I would volunteer to help solving merge conflicts.